src/components/LayoutTemplates/UNSTABLE__DetailsPage1/UNSTABLE__DetailsPage1.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file Details page template #1.
+ * @copyright IBM Security 2020
+ */
+
+import classnames from 'classnames';
+import { string, node } from 'prop-types';
+import React from 'react';
+
+import { Grid, Row } from 'carbon-components-react/lib/components/Grid';
+
+import { getComponentNamespace } from '../../../globals/namespace';
+
+export const namespace = getComponentNamespace('details-page-1');
+
+function UNSTABLE__DetailsPage1({ children, className, ...other }) {
+  const classes = classnames(namespace, className);
+  return (
+    <Grid fullWidth condensed className={classes} {...other}>
+      <Row>{children}</Row>
+    </Grid>
+  );
+}
+
+UNSTABLE__DetailsPage1.propTypes = {
+  children: node,
```

Comment:
Would you mind documenting these props for them to display in the generated Storybook 'Show Info' modal?